CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
-# 2024.xx.yy-hhmmZ
+# 2024.04.29-0600Z
+
+## Overview
+
+The current status of the libraries at the time of the release is as follows:
+
+| Library                  | Version | Status              |
+| ------------------------ | ------- | ------------------- |
+| `@dfinity/ckbtc`         | v2.3.2  | Maintained ⚙️       |
+| `@dfinity/cketh`         | v3.0.0  | Breaking Changes ⚠️ |
+| `@dfinity/cmc`           | v3.0.4  | Maintained ⚙️       |
+| `@dfinity/ic-management` | v3.2.0  | Enhanced 🔧         |
+| `@dfinity/ledger-icp`    | v2.2.3  | Maintained ⚙️       |
+| `@dfinity/ledger-icrc`   | v2.2.2  | Enhanced 🔧         |
```

Comment:
Is `Enhanced` correct? Or what is the difference between Enhanced and Maintained? Both can mean a new minor or patch version?

Reply:
Enhanced as in new feature or improvement.
Maintenaned as in "did bumped, agent-js bumped" etc. maintenance
Patch as in quick fix.

But not against better idea and wording, not that happy with it neither...

Reply:
That makes sense.

I was wondering whether it makes sense to differentiate minor and patch versions. But I don't see the extra benefit from the changelog below and specifying breaking changes.

Reply:
Sofar no one in the community asked for such a fine grain so I would say that indeed maybe not that much benefit.